homeassistant/components/device_tracker/icloud.py

```diff
@@ -0,0 +1,88 @@
+"""
+homeassistant.components.device_tracker.icloud
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Device tracker platform that supports scanning iCloud devices.
+
+It does require that your device has beend registered with Find My iPhone.
+
+Note: that this may cause battery drainage as it wakes up your device to
+get the current location.
+
+Note: You may receive an email from Apple stating that someone has logged
+into your account.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/device_tracker.icloud/
+"""
+import logging
+
+from homeassistant.const import CONF_USERNAME, CONF_PASSWORD
+from homeassistant.helpers.event import track_utc_time_change
+from pyicloud import PyiCloudService
```

Comment:
Imports should happen inside the `setup_scanner` method because requirements are not installed yet the first time this file gets loaded.